report.php

```diff
@@ -159,6 +163,15 @@
     }
 }
 
+// Define the page layout and header/breadcrumb.
+$PAGE->set_pagelayout('report');
+$PAGE->set_title(get_string('errorlogpagetitle', 'logstore_xapi'));
```

Comment:
We should use `logstorexapierrorlog` instead

Reply:
It is changed.